src/components/Tasks/tasks.css

```
@@ -137,6 +143,16 @@ input {
   box-sizing: border-box;
   outline: #2c6830;
 }
+.buddy-smaller img{
```

Tea, could you enable your prettier on saving, so that you code can get formatted automatically?
```suggestion
.buddy-smaller img {
```

---

Yes!